src/scanner/custom-rules/headers-attribute-rule.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { RuleConfiguration } from '../iruleresults';
+
+const headersAttributeCheckId: string = 'headers-attribute';
+
+export const headersAttributeRuleConfiguration: RuleConfiguration = {
+    checks: [
+        {
+            id: headersAttributeCheckId,
+            evaluate: () => true,
+        },
+    ],
+    rule: {
+        id: 'headers-attribute',
+        selector: 'th,td,[role=cell],[role=columnheader],[role=rowheader]',
```

Comment:
Is it actually correct for us to include the `role=...` cases here? As far as I can tell from the [HTML 5.2 spec](https://www.w3.org/TR/html52/tabular-data.html#the-td-element), `headers` attributes are only supported for actual `tr`/`td` elements. Do we know what screen reader support is like here?

Reply:
(if we do decide to allow `role="cell"` we should also consider whether to check `role="gridcell"` cases)

Reply:
@LiLoDavis Do you have any input on whether we should be including non-td and non-th elements with table-relevant roles for this requirement?

Reply:
> @LiLoDavis Do you have any input on whether we should be including non-td and non-th elements with table-relevant roles for this requirement?

@pownkel I agree with Dan - the `headers` attribute applies only to `<td>` and `<th>` elements, and there's no corresponding ARIA attribute.